src/components/Spinner/common/constants/index.ts

```diff
@@ -0,0 +1,7 @@
+import { SpinnerType } from '../enum'
+
+const DEFAULT_TYPE: SpinnerType = SpinnerType.PRIMARY
+const BASE_APP_COLOUR = '#1E1E1E'
```

Comment:
What does this "APP_COLOUR" mean?

Reply:
This is our main application colour according to design. I thought it good name.

Reply:
It's a good name, but it should be in a global config, not here

Reply:
Yes i take it out to global constants and change everywhere. We should do it at start project